lib/corpus.py

```diff
@@ -170,6 +170,10 @@ def segments(self):
         for sc in self.subcorpora:
             yield from sc.segments()
 
+    def all_subcorpora(self):
```

Comment:
this would be `top_level_subcorpora`, not `subcorpora`(as subcorpora can be nested).